packages/commerce-sdk-react/src/utils.ts

```diff
@@ -143,3 +144,20 @@ export const stringToBase64 =
     typeof window === 'object' && typeof window.document === 'object'
         ? btoa
         : (unencoded: string): string => Buffer.from(unencoded).toString('base64')
+
+/**
+ * Extracts custom parameters from a set of SCAPI parameters
+ *
+ * Custom parameters are identified by the 'c_' prefix before their key
+ *
+ * @param parameters object containing all parameters for a SCAPI / SLAS call
+ * @returns new object containing only custom parameters
+ */
+export const extractCustomParameters = (
+    parameters: {[s: string]: string | number | boolean | string[] | number[]} | null
```

Comment:
NIT but this is the convention we use in the commerce SDKs

```suggestion
    parameters: {[key: string]: string | number | boolean | string[] | number[]} | null
```